index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <h2>Duncan Njau</h2>
```

Comment:
- [ ] I think you should consider wrapping your title in an anchor tag `<a> `tag instead of a` </h2>` tag because the idea is to make the title clickable and always link to the homepage which is a good user experience and also semantic(good for browser accessibility) 😄.